itou/utils/migrations/0002_slyly_immutable_unaccent.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 5.1.5 on 2025-02-11 21:29
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("cities", "0001_initial"),  # Declare UnaccentExtension
+        ("utils", "0001_create_supportremark"),
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            """
+            CREATE OR REPLACE FUNCTION slyly_immutable_unaccent(text)
+                RETURNS text
+                LANGUAGE sql IMMUTABLE PARALLEL SAFE AS
+              $func$
+                SELECT public.unaccent('public.unaccent', $1)
```

Comment:
J’ai l’impression qu’on peut laisser la valeur par défaut ici :
https://www.postgresql.org/docs/current/unaccent.html#UNACCENT-FUNCTIONS

> If the dictionary argument is omitted, the text search dictionary named unaccent and appearing in the same schema as the unaccent() function itself is used.

```sql
SELECT public.unaccent($1)
```

Reply:
`Explicit is better than implicit.` non ? Après si ça te semble important ça me va aussi de changer.

Reply:
On ne spécifie pas les arguments par défaut en général (je te laisse la PR pour mettre à jour les `Form` du site :see_no_evil:). J’aurais tendance à laisser la version simple, elle fait ce qui est évident. Mais :shrug:

Reply:
Et surtout, je déclare uniquement la forme `slyly_immutable_unaccent(text)` qui du coup devrait se comporter comme `unaccent(text)` :+1: